tests/ui/infinite/infinite-instantiation-struct-tail-ice-114484.stderr

```diff
@@ -0,0 +1,86 @@
+error: reached the recursion limit finding the struct tail for `[u8; 256]`
```

Comment:
Preexisting, so if you want a good follow-up to this PR, you could pass in the obligation cause to `struct_tail_raw` and use its span for the main message and report a note for the obligation